cmd/up.go

```diff
@@ -249,9 +249,12 @@ func (up *UpContext) Execute(isRetry bool) error {
 		return err
 	}
 
-	up.Sy, err = syncthing.NewSyncthing(up.Namespace, up.Deployment.Name, devList)
-	if err != nil {
-		return err
+	runSync := up.Dev.Swap.Deployment.Container == devList[0].Swap.Deployment.Container
```

Comment:
instead of runSync I'd call this: startSyncthing or something, so it's clearer what the outcome will be